endpoints/api/namespacequota.py

```diff
@@ -94,11 +94,8 @@ def post(self, orgname):
         """
         Create a new organization quota.
         """
-        orgperm = AdministerOrganizationPermission(orgname)
```

Comment:
this change should just be @requirescope super user no need to change anything else.